File: src/main/java/io/github/wntopia/gikipedia/server/domain/auth/service/QueryAuthStatusService.java
package io.github.wntopia.gikipedia.server.domain.auth.service;

import io.github.wntopia.gikipedia.server.domain.auth.dto.response.AuthStatusResDto;
import jakarta.servlet.http.HttpSession;

public interface QueryAuthStatusService {

  AuthStatusResDto execute(HttpSession session);
}

File: src/main/kotlin/io/github/wntopia/gikipedia/server/domain/auth/service/QueryAuthStatusService.kt
package io.github.wntopia.gikipedia.server.domain.auth.service

import io.github.wntopia.gikipedia.server.domain.auth.dto.response.AuthStatusResDto
import jakarta.servlet.http.HttpSession

interface QueryAuthStatusService {
    fun execute(session: HttpSession): AuthStatusResDto
}
